docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -38,16 +74,20 @@ all the YAML files deployed in this sample to point at that namespace.
 
 1. Create a Service Account that the `ApiServerSource` runs as. The
    `ApiServerSource` watches for Kubernetes events and forwards them to the
-   Knative Eventing Broker. Create a file named `serviceaccount.yaml` and copy
-   the code block below into it.
+   Knative Eventing Broker.
+
+   {{< tabs name="create-service-account" default="By YAML" >}}
+   {{% tab name="By YAML" %}} Use following command to create the service
+   account from STDIN:
 
-   ```yaml
+   ```shell
+   cat <<EOF | kubectl create -f -
    apiVersion: v1
```

Comment:
Not sure if you want to use the `{{% readfile file="serviceaccount.yaml" %}}` shortcode here.

Reply:
oh nice, I was looking for `readfile`

Reply:
I think we should do a followup on this change